### ISSUE TITLE
Recommended Plugins: Consider adding wp-admin banner to created theme

### ISSUE THREAD

**Issue by mikachan** [WordPress/create-block-theme#410]
A "recommended plugins" input has recently been included so themers can mention plugins that work well with the created theme, however, this is only added to the readme.txt file and isn't visible anywhere else. Many other themes recommend plugins via a banner on the /themes wp-admin page. Here is an example of how Astra does it:

![image](https://github.com/WordPress/create-block-theme/assets/1645628/d2cc38e1-5445-4c36-86fc-e334bded287c)

We could consider doing something similar for created themes. I guess it would involve creating and adding a functions.php file with the relevant code.

Or is there another way we could make the recommended plugins information more visible to end users?

Originally suggested by @jamiemarsland.

**Comment by huubl** [comment]
I like the idea, but I think it would be best if "recommended plugins" (Maybe even have "Required Plugins" too) can be defined in the style.css of a theme by adding the slug of the plugin. And have the necessary functionality for handling the notifications to be built into WP core. 

`/*
Theme Name: Twenty Twenty 
Theme URI: https://wordpress.org/themes/twentytwenty/ 
[...] 
Recommended Plugins: wordpress-seo, contact-form-7 
*/`

**Comment by carolinan** [comment]
I don't think we should encourage themes to add these adverts, especially not before there is a notification center.

This is the core ticket for theme and plugin dependencies: https://core.trac.wordpress.org/ticket/22316

**Comment by mikachan** [comment]
Thanks, both, for your input here. I agree that it makes more sense to include this in Core.

> This is the core ticket for theme and plugin dependencies: https://core.trac.wordpress.org/ticket/22316

This is super helpful, thank you! Watched ✅
